### PR TITLE
Add Go examples

### PR DESCRIPTION
**Pull request by dangeross**
~~Let's wait until the next SDK release to generate the package and publish docs~~

### CODE REVIEW

File: theme/highlight.js [hunk 1]
```diff
@@ -7,20 +7,20 @@ var hljs=function(){"use strict";function e(n){
 return n instanceof Map?n.clear=n.delete=n.set=()=>{
 throw Error("map is read-only")}:n instanceof Set&&(n.add=n.clear=n.delete=()=>{
 throw Error("set is read-only")
-}),Object.freeze(n),Object.getOwnPropertyNames(n).forEach((t=>{
-const s=n[t],a=typeof s;"object"!==a&&"function"!==a||Object.isFrozen(s)||e(s)
+}),Object.freeze(n),Object.getOwnPropertyNames(n).forEach((s=>{
```
Comment (roeierez) [inline theme/highlight.js:10]: @dangeross  Do we need to commit this file?

Comment (roeierez) [reply theme/highlight.js:10]: Is this because you added the golang syntax?

Comment (dangeross) [reply theme/highlight.js:10]: I think it is needed for the dart syntax highlighting: https://github.com/breez/breez-sdk-docs/commit/680f9b7b569923b20798002b86e1483ff1bbb13c

But I don't know why it's committed changes here

**Review by roeierez — APPROVED** [review]

File: theme/highlight.js (hunk 1 above)
Comment (dangeross) [reply theme/highlight.js]: Was a merge issue, fixed

**Pull request merged**